### ISSUE TITLE
latf-load 3.0.0 does not exit with errors and count warning as error

### ISSUE THREAD

**Issue by komstat** [ncbi/sra-tools#601]
We deployed latf-load 3.0.0 to our submission system. However, there seems to be bugs.

1. latf-load 3.0.0 does not exit with errors
When errors exceed allowed limit, latf-load 2.9.6 exits with errors. However, 3.0.0 does not exit with errors. These silent errors make our submission processes stack.

2. latf-load 3.0.0 count warnings as errors
3.0.0 counts warnings as errors. This cause more errors in our submission processes and in addition to the bug 1, this also makes our submission processes stack.

We appreciate if you could fix these bugs as soon as possible.

**Comment by komstat** [comment]
We executed this binary executable.  
https://ftp-trace.ncbi.nlm.nih.gov/sra/sdk/3.0.0/sratoolkit.3.0.0-centos_linux64.tar.gz   

Here is the command.  
```
sratoolkit.3.0.0-centos_linux64/bin/latf-load --platform ILLUMINA --quality PHRED_33  -o test test3.fq.gz
```

After outputting 1,001 warnings to stderr, 
latf-load outputs "error excessive while reading file within alignment module - Number of errors 1001 exceeds limit of 1000: Exiting", 
but the tool does not exit correctly.   
```
2022-03-08T23:44:51 latf-load.3.0.0 warn: data inconsistent while reading file within application support module - Spot 'm180206_022314_42208_c101407522550000001823289503161857_s1_p0', which was first seen with mate info, now has no mate info
2022-03-08T23:44:52 latf-load.3.0.0 warn: data inconsistent while reading file within application support module - Spot 'm180206_022314_42208_c101407522550000001823289503161857_s1_p0', which was first seen with mate info, now has no mate info
.... warn: data inconsistent .....
2022-03-08T23:42:20 latf-load.3.0.0 warn: data inconsistent while reading file within application support module - Spot 'm180206_022314_42208_c101407522550000001823289503161857_s1_p0', which was first seen with mate info, now has no mate info
2022-03-08T23:42:20 latf-load.3.0.0 err: error excessive while reading file within alignment module - Number of errors 1001 exceeds limit of 1000: Exiting
```

Here is the test fastq.gz.  
https://www-dev.ddbj.nig.ac.jp/test3.fq.gz

We run the tool in "CentOS Linux release 7.5.1804 (Core)".

**Comment by aboshkin** [comment]
@komstat thank you! We have reproduced the problem and are investigating.

**Comment by aboshkin** [comment]
@komstat could you please post the test input file again (https://www-dev.ddbj.nig.ac.jp/test3.fq.gz)? Thank you!

**Comment by komstat** [comment]
@aboshkin I have re-uploaded the file. Thank you.

**Comment by aboshkin** [comment]
@komstat thank you! We are working on a fix, should be in the next release.

**Comment by klymenko** [comment]
@komstat, it was fixed in 3.0.1 release.
Please check.
